src/mito2/src/memtable/merge_tree/data.rs

```diff
@@ -94,23 +108,302 @@ impl DataBuffer {
     }
 
     /// Freezes `DataBuffer` to bytes. Use `pk_lookup_table` to convert pk_id to encoded primary key bytes.
```

Comment:
```suggestion
    /// Freezes `DataBuffer` to bytes. Use `pk_weights` to convert pk_id to encoded primary key bytes.
```

Reply:
fixed in https://github.com/evenyag/greptimedb/pull/8/commits/5f6cabc794a1e8b38d9f55d06523b61ca06c883e